tree/30_generic_methods/_log_default.cf

```diff
@@ -0,0 +1,74 @@
+#####################################################################################
+# Copyright 2013-5 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Log default
+# @description Standard default logging output
+#
+# @parameter message              The common part of the message to display
+# @parameter old_class_prefix     The prefix of the class for different states (0.x version)
+# @parameter origin_class_prefix  The prefix of the class for different states (1.x version)
+# @parameter args                 The arguments used to call the generic method (slist)
+#
+# @class_prefix _logger_default
```

Comment:
> why the class prefix ? i'm pretty sure there's none of them

All bundles must have a class_prefix. It's our internal style tests' policy. If we don't have one, tests will fail.

Also, there has always been one in this file, I just copied it without changing this line (see the individual commits, rather than the general overview)